### DISCUSSION TITLE
HOW TO SWITCH OFF OBS STUDIO WITH ADVANCED SCENE SWITCHER PLUGIN

### DISCUSSION THREAD

**Discussion by polin79** [WarmUpTill/SceneSwitcher#1165] [Q&A] [answered]
Hi,
Is there an Action in Advanced Scene Switcher plugin to exit and turn-off OBS Studio at the end of a specific Macro?

Thanks
Paolo

**Comment by WarmUpTill** [comment, marked as answer]
Hello,
yes, this is possible using the "Plugin state" action type:
![image](https://github.com/user-attachments/assets/cd9a9daa-2464-4fd7-92c0-fa202b119ca9)

**Reply by polin79** [reply]
Thank you very very much for the support.

Paolo
